blinkpy/camera.py

```diff
@@ -350,50 +354,48 @@ def expire_recent_clips(self, delta=datetime.timedelta(hours=1)):
             for clip in self.recent_clips:
                 url = clip["clip"]
                 if "local_storage" in url:
-                    api.http_post(self.sync.blink, url)
+                    await api.http_post(self.sync.blink, url)
 
-    def get_liveview(self):
+    async def get_liveview(self):
         """Get livewview rtsps link."""
-        response = api.request_camera_liveview(
+        response = await api.request_camera_liveview(
             self.sync.blink, self.sync.network_id, self.camera_id
         )
         return response["server"]
 
-    def image_to_file(self, path):
+    async def image_to_file(self, path):
         """
         Write image to file.
 
         :param path: Path to write file
         """
         _LOGGER.debug("Writing image from %s to %s", self.name, path)
-        response = self.get_media()
-        if response.status_code == 200:
+        response = await self.get_media()
+        if response.status == 200:
             with open(path, "wb") as imgfile:
-                copyfileobj(response.raw, imgfile)
+                copyfileobj(await response.raw(), imgfile)
```

Comment:
Assuming I did this right - any message buffers (`.text()', `.json()`, etc) are async calls to return an `str` rather than actually being a string. It's good to question everything - I'm sure I've missed some of the conversions.